docs/data/stashcache/run-stash-origin-container.md

```diff
@@ -49,6 +51,25 @@ XC_RESOURCENAME=YOUR_SITE_NAME
 ORIGIN_FQDN=<FQDN>
 ```
 
+### Ensure the xrootd service has a certificate
+
+The service will need a certificate for contacting central OSDF services.
+
+The certificate and key must be inside the container at `/etc/grid-security/xrd/xrd{cert,key}.pem`,
+owned by the `xrootd` user and group (UID/GID 10940).
+
+The easiest solution for this is to use your host certificate and key.
+Volume-mount the certificate to `/etc/grid-security/hostcert.pem` and the key to `/etc/grid-security/hostkey.pem`.
+The files will be copied with the right permissions on container startup.
+
+!!! note
+    You must restart the container whenever you renew your certificate
+    in order for the services to pick up the new certificate.
+    If you automate certificate renewal, you should automate restarts as well.
+    For example, if you are using Certbot for Let's Encrypt, you should write a "deploy hook" as documented
+    [on the Certbot site](https://certbot.eff.org/docs/using.html#renewing-certificates).
```

Comment:
Separately, maybe we should add a cron that runs the copy periodically?

Reply:
We do the copy on container startup; since we tell people to kick the container when their cert gets updated (instead of telling supervisor to restart xrootd), that's all that's necessary.

Reply:
Oh, so even if they updated the cert on disk and we picked it up, XRootD needs the restart to recognize it? :(

Reply:
Yeah I think XRootD just loads the cert/key once on startup and doesn't look at it again.